pp/series_index/trie/cedarpp_tree.h

```diff
@@ -126,7 +122,13 @@ class CedarTrie {
   using EnumerativeIterator = CedarEnumerativeIterator;
   using Value = uint32_t;
 
-  PROMPP_ALWAYS_INLINE void insert(std::string_view key, uint32_t value) noexcept { trie_.update(key.data(), key.length(), 0) = value; }
+  PROMPP_ALWAYS_INLINE void insert(std::string_view key, uint32_t id) noexcept {
+    if (max_id_ == kNoId || max_id_ < id) {
```

Comment:
have you benchmark - what is faster: two condition or increment with condition? Maybe using length has better performance than max_id with initial constant?